docs/EDID to Receiver Caps Mapping.md

```diff
@@ -61,11 +61,13 @@ Standard Timing Definitions (STD) format is defined in [E-EDID][E-EDID] section
 - `urn:x-nmos:cap:format:grain_rate` MUST be calculated with _Field Refresh Rate_
 - `urn:x-nmos:cap:format:interlace_mode` MUST be set to `progressive`
 
+Some of STDs are described in [DMT][DMT] with more precise vertical frequencies than _Field Refresh Rate_. Such STDs SHOULD be described with `urn:x-nmos:cap:format:grain_rate` supporting both frame rate values.
+
 [Example](./Examples.md#standard-timings)
 
 #### Detailed Timing Descriptors (18 Byte Descriptors)
 
-Defined in [E-EDID][E-EDID] section 3.10, there are 4 possible descriptors which can be provided. The first, _Preferred Timing Mode_ is an obligation and MUST have a `urn:x-nmos:cap:meta:preference` value of `100`. Any Detailed Timing Descriptors in [CTA-861][CTA-861] Extension Block (section 7.2.1) MUST follow the mapping.
+Defined in [E-EDID][E-EDID] section 3.10, there are 4 possible descriptors which can be provided. The first, _Preferred Timing Mode_ is an obligation and MUST have the highest `urn:x-nmos:cap:meta:preference` value amid the Constraint Sets. Detailed Timing Descriptors describing _Native Video Formats_ MUST have higher `urn:x-nmos:cap:meta:preference` values than video modes not marked as native. Any Detailed Timing Descriptors in [CTA-861][CTA-861] Extension Block (section 7.2.1) MUST follow the mapping.
```

Comment:
Obligation... does this mean a requirement of E-EDID? From our mapping point of view could we talk about it being "always present"?

I think the MUST here need to have an "unless overridden by the VIC case describes below" or something??

```suggestion
Defined in [E-EDID][E-EDID] section 3.10, there are 4 possible descriptors which can be provided. The first, _Preferred Timing Mode_, is always present and the Constraint Set(s) that represent it MUST have the highest `urn:x-nmos:cap:meta:preference` value among the Constraint Sets. Constraint Sets for Detailed Timing Descriptors describing _Native Video Formats_ MUST have higher `urn:x-nmos:cap:meta:preference` values than video modes not marked as native. Any Detailed Timing Descriptors in [CTA-861][CTA-861] Extension Block (section 7.2.1) MUST follow the mapping.
```

Reply:
Yes, you're right, `always present` is better.

The language connecting this case with the VIC case looks better than anything that appeared in my head. I also thought about extracting `preference` rules to a separate section.